src/NotSureWhere/parseLaMEM.jl

```diff
@@ -0,0 +1,27 @@
+using LaMEM, GeophysicalModelGenerator
+
+# structure definition
+struct markerGrid
+    Xpart::Array{Float64,3}
+    Ypart::Array{Float64,3}
+    Zpart::Array{Float64,3}
+    Temp::Array{Float64,3}
+    Phase::Array{Int64,3}
+end
```

Comment:
```suggestion
struct markerGrid
    Xpart::Array{Float64,3}
    Ypart::Array{Float64,3}
    Zpart::Array{Float64,3}
    Temp::Array{Float64,3}
    Phase::Array{Int64,3}
end
```
 Two things here
1. As a general convention, `structs` names start with upper case. I would call it `MarkerGrid`
2. It's usually better to parameterize the structs, so that they more general and you get some compile-time information that can be useful later for multiple-dispatch or other optimizations. For example:
```julia
struct markerGrid{N, T, I}
    Xpart::Array{T, N}
    Ypart::Array{T, N}
    Zpart::Array{T, N}
    Temp::Array{T, N}
    Phase::Array{I, N}
end
```

Reply:
ok